lib/dev/ui/spinner.rb

```diff
@@ -40,6 +40,7 @@ def initialize(title, &block)
               end
             end
 
+            @force_full_render = false
```

Comment:
It's probably a bad idea to let the task control render timing, since then you can get jittering/inconsistency between different items in the spingroup.